### PR TITLE
feat: added ratings

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the rating component.

image

### CODE REVIEW

File: stylesheets/components/ratings/ratings.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import "/stylesheets/index.css";
+
+.star{
+    color: var(--default-grey);
+}
+
+.ratings-after .star{
+    color: gold;
+}
```
Comment (ShrutiLakhani) [inline stylesheets/components/ratings/ratings.css:9]: Clean Code. Good to merge

**Review by ShrutiLakhani — APPROVED** [review]
GTM!

**Review by m26unkwn — APPROVED** [review]
GTM!

Everything is good to me. .

File: stylesheets/colors.css [hunk 2]
```diff
@@ -72,4 +74,9 @@
     --grid-2-section: #fef2f2;
     --grid-3-section: #fdf2f8;
     --grid-wrapper: #fafafa;
+
+    --headings-section:  #f0e3e3;
```
Comment (manojsarna) [inline stylesheets/colors.css:78]: can use better color variable naming convention. eg --COLOR-BLACK , in VSCODE while typing --COLOR all the color list will populate!! :)

File: stylesheets/documentation.css [hunk 3]
```diff
@@ -77,6 +77,19 @@
...
+    margin: auto;
+    border-radius: 7px;
+ }
+ 
+ .class-highlight{
+    background: var(--code-highlight-bg);
+    padding: 5px;
+    border-radius: 5px;
+ }
+
```
Comment (manojsarna) [inline stylesheets/documentation.css:92]: rest of the code looks good!!

File: docs/ratings.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            <a href="/docs/cards.html" class="nav-subtitles">Cards</a>
+            <a href="/docs/images.html" class="nav-subtitles">Images</a>
+            <a href="/docs/inputs.html" class="nav-subtitles">Input</a>
+        </nav>
+
+        <section class="content ratings-section">
+
+            <div class="component-wrapper">
+                <h1 class="wrapper-heading">Ratings</h1>
+                <hr>
```
Comment (manojsarna) [inline docs/ratings.html:40]: check the spacng be consistent with it

File: docs/ratings.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+                    <br>
+                </div>
+
+                <div class="component-showcase-col ratings-after">
+                    <h3 class="rating-before">After Rating:</h3>
+                    <div class="stars-container">
+
+                        <span class="star filled-star material-icons md-36 material-icons-outlined">
+                            star
+                        </span>
```
Comment (manojsarna) [inline docs/ratings.html]: here also check

File: docs/ratings.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        </section>
+
+    </div>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"
+        integrity="sha512-IaaKO80nPNs5j+VLxd42eK/7sYuXQmr+fyywCNA0e+C6gtQnuCXNtORe9xR4LqGPz5U9VpH+ff41wKs/ZmC3iA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script>
+        hljs.highlightAll();
+    </script>
```
Comment (manojsarna) [inline docs/ratings.html]: code can be beautify over here

**Review by manojsarna — APPROVED** [review]
Overall code is good

can be merged!!

**Comment by SinchanaMS** [comment]
Made PR Changes. Merging to dev.

**Pull request merged**